### PR TITLE
Remove idleTimeout and increase ping interval to 20 seconds.

### PR DESCRIPTION
**Pull request by ydidwania**

### CODE REVIEW

**Review by djmitche — APPROVED** [review]
The README changes look good to me.  I'm in a rush so I didn't look at the code :)

File: test/getmsg_test.js [hunk 1]
```diff
@@ -40,30 +40,4 @@ helper.secrets.mockSuite(__filename, [], function(mock, skipping) {
...
-    evtSource.addEventListener('error', (err) => {
-      evtSource.close();
-      assert(JSON.parse(err.data) === 'No messages received for 20s. Aborting...');
-      pass();
-    });
-
-    await resolve;
-  });
-
 });
```
Comment (helfi92) [inline test/getmsg_test.js:43]: In L67,

```js
if (req.headers['last-event-id']) {
  debug('no reconnects allowed');
  return res.reportError('NoReconnects', 'Not allowing reconnects');
}
```

Shouldn't the if statement do a check against the `-`? Something like:

```js
if (req.headers['last-event-id'] === '-') {
 // ...
}
```

Comment (helfi92) [inline test/getmsg_test.js:43]: There's a couple of places where we use `json_bindings`. Let's change those to `jsonBindings` to respect JS conventions. Apologies for not catching this earlier.

Comment (ydidwania) [reply test/getmsg_test.js:43]: We set the id of all messages to `-` .  
Only the initial request doesn't have the `last-event-id` header set

**Review by imbstack — APPROVED** [review]
What was the original justification for `idleTimeout`?

**Review by helfi92 — APPROVED** [review]

**Comment by ydidwania** [comment]
@imbstack I kept the idleTimeout so that cleints cannot make requests with a random `routingKey` which never receives a messge.
An incorrect `exchange` returns an error, but there's no way to detect incorrect `routingKeyPattern` other than no messages for a very long time

File: test/helper.js [hunk 2]
```diff
@@ -71,9 +71,9 @@ exports.withServer = (mock, skipping) => {
     exports.load.cfg('taskcluster.rootUrl', helper.rootUrl);
 
     helper.connect = bindings => {
-      let json = urlencode(JSON.stringify(bindings));
+      let jsonBindings = urlencode(JSON.stringify(bindings));
       debug('Connecting to api...');
-      const evtSource = new EventSource(libUrls.api(helper.rootUrl, 'events', 'v1', '/connect/?bindings=')+json);
+      const evtSource = new EventSource(libUrls.api(helper.rootUrl, 'events', 'v1', '/connect/?bindings=')+jsonBindings);
```
Comment (helfi92) [inline test/helper.js]: Are you able to use template strings for the last argument?

```js
const evtSource = new EventSource(libUrls.api(helper.rootUrl, 'events', 'v1', `/connect/?bindings=${jsonBindings}`));
```

Comment (ydidwania) [reply test/helper.js]: Yes. Done!

**Review by helfi92 — APPROVED** [review]

**Pull request merged**